monai/metrics/panoptic_quality.py

```diff
@@ -274,7 +274,7 @@ def _get_paired_iou(
 
         return paired_iou, paired_true, paired_pred
 
-    pairwise_iou = pairwise_iou.cpu().numpy()
```

Comment:
typehint not work here since:
```
monai/metrics/panoptic_quality.py:271:50: error: Argument 1 to "nonzero" has incompatible type "NdarrayTensor"; expected "Tensor"  [arg-type]
monai/metrics/panoptic_quality.py:278:20: error: Item "ndarray[Any, Any]" of the upper bound "ndarray[Any, Any] | Tensor" of type variable "NdarrayTensor" has no attribute "cpu"  [union-attr]
monai/metrics/panoptic_quality.py:278:20: error: Incompatible types in assignment (expression has type "Any | ndarray[Any, Any]", variable has type "NdarrayTensor")  [assignment]
```